addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -122,6 +122,15 @@ export class PropertiesField extends Component {
         return false;
     }
 
+    /**
+     * Generate a unique ID to be used in the DOM.
+     *
+     * @returns {string}
+     */
+    getUniqueDomID() {
```

Comment:
Could we align both and make this a property?

So in this and in "property_definition" you would have a:

```js
get uniqueDomID() {
   return `property_${uuid()}`;
}
```

And in definition you would use it and append a suffix to it.
That way we have similar logic between the two.

Reply:
IMO it shouldn't be a getter here, because a new value is generated each time we call the method (so it will not be clear if we use a getter)

but I can use a method for the definition which take the suffix in argument (so we won't need to build the string in the template)